### DISCUSSION TITLE
Installation plugin issue

### DISCUSSION THREAD

**Discussion by angelfish0808** [jupyter/notebook#7282] [Q&A]
Can plugins be installed for versions greater than 7.0 and how to install them，thanks!

**Comment by jtpio** [comment]
@angelfish0808 you can install extensions for Notebook 7 in the same way as for JupyterLab, using `pip` or `conda`: https://jupyter-notebook.readthedocs.io/en/stable/notebook_7_features.html#support-for-many-jupyterlab-extensions

For example to install to install the `jupyterlab-night` theme: `pip install jupyterlab-night`. And then reload the page.

**Reply by angelfish0808** [reply]
What I understand is that the installed extensions can be used in both Jupyterlab and jupyternotebook. Is this what you mean?
